core/dataURI/dataURI.go

```diff
@@ -65,9 +71,9 @@ func AppendData(uri string, dataReader io.Reader, dataLength uint32, offset int6
 
 	if isLastChunk && !isTempData {
 		if trace.IsLogging(logger.TRACE) {
-			trace.Trace("Rename file from %s to %s", filePath, dataURI.Path)
+			trace.Trace("Rename file from %s to %s", tmpFilePath, dataURI.Path)
 		}
-		if err := os.Rename(filePath, dataURI.Path); err != nil {
+		if err := os.Rename(tmpFilePath, dataURI.Path); err != nil {
```

Comment:
Should this be `os.Rename(tmpFilePath, dataURI.Path)` or `os.Rename(tmpFilePath, baseFilePath)`?